packages/contracts-periphery/contracts/universal/op-erc721/OptimismMintableERC721Factory.sol

```diff
@@ -52,9 +50,6 @@ contract OptimismMintableERC721Factory is Semver, OwnableUpgradeable {
     function initialize(address _bridge, uint256 _remoteChainId) public initializer {
         bridge = _bridge;
         remoteChainId = _remoteChainId;
```

Comment:
Interesting, what's this used for?

Reply:
We pass it in as an argument when we create an `OptimismMintableERC721` contract [here](https://github.com/ethereum-optimism/optimism/pull/3459/files/c08151919438fc6dd062a9e14bc323ddd863eb77#diff-d3bf3abced4d6814b0b05ab3682e7bf16a8a8f9de683b07e8fc244fbf7997274R79). It's used in the `OptimismMintableERC721` [here](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-periphery/contracts/universal/op-erc721/OptimismMintableERC721.sol#L64) to create the EIP-681 URI.